app/src/main/java/de/danoeh/antennapod/fragment/ChaptersFragment.java

```diff
@@ -43,11 +45,26 @@ public class ChaptersFragment extends AppCompatDialogFragment {
     @NonNull
     @Override
     public Dialog onCreateDialog(@Nullable Bundle savedInstanceState) {
-        return new MaterialAlertDialogBuilder(requireContext())
+
+        AlertDialog dialog = new MaterialAlertDialogBuilder(requireContext())
                 .setTitle(getString(R.string.chapters_label))
                 .setView(onCreateView(getLayoutInflater()))
-                .setNegativeButton(getString(R.string.close_label), null) //dismisses
-                .create();
+                .setPositiveButton(getString(R.string.close_label), null) //dismisses
+                .setNeutralButton("Reset", null)
+                .show();
+        dialog.show();
+        dialog.getButton(DialogInterface.BUTTON_NEUTRAL).setOnClickListener(v -> {
```

Comment:
Hmm, this feels like a hacky workaround. I would prefer to not call `.show()` in `onCreateDialog`. There must be another method that is called when the dialog is actually shown.